homeassistant/components/gios/const.py

```diff
@@ -35,41 +32,48 @@
 ATTR_SO2: Final = "so2"
 ATTR_AQI: Final = "aqi"
 
-SENSOR_TYPES: Final[dict[str, SensorDescription]] = {
-    ATTR_AQI: {},
-    ATTR_C6H6: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_CO: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_NO2: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_O3: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_PM10: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_PM25: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-    ATTR_SO2: {
-        ATTR_UNIT: CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-        ATTR_STATE_CLASS: STATE_CLASS_MEASUREMENT,
-        ATTR_VALUE: round,
-    },
-}
+SENSOR_TYPES: Final[tuple[GiosSensorEntityDescription, ...]] = (
+    GiosSensorEntityDescription(key=ATTR_AQI, name="AQI", value=None),
```

Comment:
```suggestion
    GiosSensorEntityDescription(key=ATTR_AQI, name="AQI", value=None,),
```
Can you add a trailing comma here? Black will then move each key-value pair to a new line.

Reply:
But why if it fits on one line? Do you want to keep formatting consistent?

Reply:
Yeah, exactly. It's a bit easier to read as well, at least IMO.

Reply:
Yeah, you're right, it's now more readable.